src/ledger/LedgerTxnAccountSQL.cpp

```diff
@@ -209,6 +250,7 @@ LedgerTxnRoot::Impl::insertOrUpdateAccount(LedgerEntry const& entry,
     st.exchange(soci::use(entry.lastModifiedLedgerSeq, "v8"));
     st.exchange(soci::use(liabilities.buying, liabilitiesInd, "v9"));
     st.exchange(soci::use(liabilities.selling, liabilitiesInd, "v10"));
+    st.exchange(soci::use(signers, "v11"));
```

Comment:
you should set this to `NULL` when `signers` is empty.
Look how it's done with `st.exchange(soci::into(inflationDest, inflationDestInd));`